### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Lysautumn**
Changed some wording throughout for clarity. Did not edit sections about command line and more technical issues (like branching), which will probably need more discussion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
 ### Install Git
-Here is a detailed explanation of [how to install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+Here is a detailed explanation of [how to install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
 
 ### Fork the project
-A fork is a copy of a project. In this case, you will be copying the project [software-interview-prep](https://github.com/approachable-io/software-interview-prep), making a change to that copy, and then making a Pull Request - a request for the original product to incorporate your changes.
+A fork is a copy of a project. In this case, you will be copying the project [software-interview-prep](https://github.com/approachable-io/software-interview-prep), make a change to that copy, and then make a Pull Request. A Pull Request is a request for the original project to include your changes.
```
Comment (redxtech) [inline README.md]: The changes here don't really make any sense. "In this case, you will be copying the project, make a change to that copy, and then make a pull request...". The wording for that was fine before. The description of a pull request is good though.

File: README.md [hunk 2]
```diff
@@ -88,11 +88,7 @@ git checkout <branch name>
 
 ### Making your changes
 
-Now that we have our own branch, we can begin to contribute. First, we need to make some changes. What I'm doing 
-right now, is adding images to the `README.md` file, so I would open up any text editor, and open the file.
-
-Any editor or IDE will work just fine (I personally use WebStorm for projects and sublime for one-off files), Some 
-popular editors include Atom, VS Code, and even vim or nano if you have experience using either.
+Now that we have our own branch, we can make our changes. There are two ways to edit files, either in GitHub, or with a text editor on your computer. Open the file containing the code in a code editor. You can use any code editing program you are comfortable with, popular choices are Atom, VS Code, and Sublime Text.
```
Comment (redxtech) [inline README.md]: This part here doesn't really add much at all.

File: README.md (hunk 1 above)
Comment (LukeSchlangen) [reply README.md]: I agree with @redxtech, I think "making" a change is correct grammatically, but I like the Pull Request description.

File: README.md (hunk 2 above)
Comment (LukeSchlangen) [reply README.md]: I think this section could probably use a lot more explicit details. I know we're trying to be agnostic on technology here, but I don't think that agnosticism is more important than clarity for our first-time users. I think this section will need a pretty large addition (maybe in supplemental documentation) in order to help truly new users who have never used git or written code to the point where they can add a new interview question.

**Comment by Lysautumn** [comment]
@LukeSchlangen I have made the changes that you requested to the "Fork the project" section.

**Comment by LukeSchlangen** [comment]
@Lysautumn Thank you for your contribution!

**Pull request merged**